README.md

```diff
@@ -19,7 +19,42 @@ using AbstractDifferentiation
 
 ## `AbstractDifferentiation` backends
 
-To use `AbstractDifferentiation`, first construct a backend instance `ab::AD.AbstractBackend` using your favorite differentiation package in Julia that supports `AbstractDifferentiation`. For higher order derivatives, you can build higher order backends using `AD.HigherOrderBackend`. For instance, let `ab_f` be a forward-mode automatic differentiation backend and let `ab_r` be a reverse-mode automatic differentiation backend. To construct a higher order backend for doing forward-over-reverse-mode automatic differentiation, use `AD.HigherOrderBackend((ab_f, ab_r))`. To construct a higher order backend for doing reverse-over-forward-mode automatic differentiation, use `AD.HigherOrderBackend((ab_r, ab_f))`.
+To use `AbstractDifferentiation`, first construct a backend instance `ab::AD.AbstractBackend` using your favorite differentiation package in Julia that supports `AbstractDifferentiation`.
+The following backends are made available by `AbstractDifferentiation` if their corresponding package is loaded (thanks to [Requires.jl](https://github.com/JuliaPackaging/Requires.jl)):
```

Comment:
FYI this is only a workaround until the API is stabilized. Ideally these backends will be defined in the respective AD packages.

Reply:
Good to know! In the meantime I still think it can help users to know what's available without digging into the source code of AbstractDifferentiation.jl as I did.

Reply:
Sure, I just wanted to point out that this will change at some point and Requires will be dropped then.

Reply:
```suggestion
The following backends are temporarily made available by `AbstractDifferentiation` as soon as their corresponding package is loaded (thanks to [Requires.jl](https://github.com/JuliaPackaging/Requires.jl)).
In the long term, these backend objects are meant be defined within their respective packages to enforce the `AbstractDifferentiation` interface.
```